lxc/list.go

```diff
@@ -118,6 +121,9 @@ lxc list -c ns,user.comment:comment
 
 const defaultColumns = "ns46tSL"
 
+// Cache the GetImageServer lookup
+var localServer lxd.ImageServer
```

Comment:
This should really be done against the same server as `lxc list` itself is running against.